dataset/dictionary/weapons.json5

```
@@ -1417,7 +1417,7 @@
     en: "Memory of Dust",
     ja: "浮世の錠",
     zhCN: "尘世之锁",
-    pronunciationJa: "うきよのじょう", // TODO Need check
+    pronunciationJa: "うきよのじょう",
```

Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "うきよのじょう", // TODO Need Check
```